plugins/modules/ovirt_vm_os_info.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
```

Comment:
Hmm, is there a new preferred header for Ansible collection modules? Because collections are not longer part of Ansible. For now it's OK here, but please file an issue to ansible/ansible to get this resolved and documented, so we can update headers in the whole ovirt collection

Reply:
Why create an issue on ansible/ansible should we fix it here? I'm probably missing something.

Reply:
I've just asked the question in #ansible-devel so hopefully will get an answer in there.

Reply:
@mwperina @mnecas its been added to existing collections issue as advised by @gundalow on IRC. Also advised it shouldn't be a blocker for continuing development.

Reply:
> Why create an issue on ansible/ansible should we fix it here? I'm probably missing something.

Ansible team should define guidelines for collections, that's why I suggested to create issue on ansible/ansible to provide such guidelines

> @mwperina @mnecas its been added to existing collections issue as advised by @gundalow on IRC. Also advised it shouldn't be a blocker for continuing development.

Of course, I haven't suggested to wait until this is resolved, just to raise that. Thanks!